web-components/src/wc_scss/themes/global--mdv2.scss

```diff
@@ -1,6 +1,11 @@
+@import "@/wc_scss/typography/settings.scss";
+
+$brand-font-regular: "Inter Regular", $font-family-sans-serif;
+
 @mixin global-theme-specfic-vars {
   --rem-base: 14px;
   font-size: var(--rem-base);
+  font-family: $brand-font-regular;
 
```

Comment:
To get this to work you could add a css variable for example

```
--brand-font-regular
```

Then in each component update them to use 

font-family: var(--brand-font-regular, $brand-font-regular;)